src/bundle/DependencyInjection/Compiler/RichTextHtml5ConverterPass.php

```diff
@@ -37,6 +37,14 @@ public function process(ContainerBuilder $container)
             );
             $this->setConverterDefinitions($taggedInputServiceIds, $html5InputConverterDefinition);
         }
+
+        if ($container->hasDefinition('ibexa.richtext.converter.edit.xhtml5')) {
+            $html5EditConverterDefinition = $container->getDefinition('ibexa.richtext.converter.edit.xhtml5');
+            $taggedInputServiceIds = $container->findTaggedServiceIds(
+                'ibexa.field_type.richtext.converter.edit.xhtml5'
+            );
+            $this->setConverterDefinitions($taggedInputServiceIds, $html5EditConverterDefinition);
+        }
```

Comment:
Why do you need this?

Reply:
The richtext->xhtml5edit transformation does not currently use the Aggregate converter (only one fixed converter was used), so I had to fix that in order to support on-the-fly conversion